evap/evaluation/models.py

```diff
@@ -1111,9 +1111,11 @@ def send_to_users_in_courses(cls, template, courses, recipient_groups, use_cc, r
     def send_to_user(cls, user, template, subject_params, body_params, use_cc, request=None):
         if not user.email:
             warning_message = "{} has no email address defined. Could not send email.".format(user.username)
-            logger.warning(warning_message)
             if request is not None:
+                logger.warning(warning_message)
                 messages.warning(request, _(warning_message))
+            else:
+                logger.error(warning_message)
```

Comment:
please add one or two comments explaining both what `if request is not None` means and why this is a warning or an error depending on that.